src/Tasks/Microsoft.NET.Build.Tasks/msbuildExtensions/Microsoft.NET.Build.Extensions/Microsoft.NET.Build.Extensions.ConflictResolution.targets

```diff
@@ -0,0 +1,55 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Choose>
+    <!-- Allow completely disabling the conflict resolution targets -->
+    <When Condition="'$(DisableHandlePackageFileConflicts)' == 'true'" />
+    <!-- SDK projects have built in conflict resolution that understands assets/deps file -->
+    <When Condition="'$(UsingMicrosoftNETSdk)' == 'true'" />
+    <When Condition="'$(TargetFramework)' != '' or '$(TargetFrameworks)' != ''">
+      <!-- SDK project missing inbox conflict resolution.  Emit an error -->
+      <PropertyGroup>
+        <PackagingToolsInsufficientSDKVersionAfter>ResolvePackageDependenciesForBuild</PackagingToolsInsufficientSDKVersionAfter>
+      </PropertyGroup>
+    </When>
+    <When Condition="'$(ResolveNuGetPackages)' == 'true' and Exists('$(ProjectLockFile)')">
+      <!-- Non-SDK using project.json or PackageReference, run after references are read from the lock/assets file -->
+      <PropertyGroup>
+        <HandlePackageFileConflictsAfter>ResolveNuGetPackageAssets</HandlePackageFileConflictsAfter>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <!-- No specific NuGet targets to sequence after, run before targets that consume references -->
+      <PropertyGroup>
+        <ResolveAssemblyReferencesDependsOn>$(ResolveAssemblyReferencesDependsOn);HandlePackageFileConflicts</ResolveAssemblyReferencesDependsOn>
+        <PrepareResourcesDependsOn>HandlePackageFileConflicts;$(PrepareResourcesDependsOn)</PrepareResourcesDependsOn>
+      </PropertyGroup>
+    </Otherwise>
+  </Choose>
+
+  <Target Name="PackagingToolsInsufficientSDKVersion" Condition="'$(PackagingToolsInsufficientSDKVersion)' != 'false'" AfterTargets="$(PackagingToolsInsufficientSDKVersionAfter)">
+    <Error Text="The version of Microsoft.NET.Sdk used by this project is insufficient to support this package.  Please ensure you are using the .NET Core SDK 2.0 or higher, which is included in Visual Studio 2017 Update 15.3.  This error may be suppressed by setting PackagingToolsInsufficientSDKVersion=false." />
```

Comment:
We can't make it work on 1.1 SDK without bringing back deps file re-writing.  We discussed this and decided to raise the error in the NETStandard injection targets, only when netstandard is referenced and running on old SDK.